tensorflow/stream_executor/dnn.h

```diff
@@ -876,6 +876,22 @@ enum class ElementwiseOperation { kAdd, kMultiply };
 
 string ElementwiseOperationString(ElementwiseOperation op);
 
+// A simple class to workaround the too perfect forwarding issue at
```

Comment:
Some minor nits:
  * Change "A simple class to..." to something like "A simple class representing the version of the backing library, to..."
  * Put "too perfect forwarding" in quotes
  * Change "at gcc6+ compilers" to "in gcc 6+"
  * End sentence in period.